src/main/java/com/arcbees/analytics/client/ClientAnalytics.java

```diff
@@ -71,48 +100,125 @@ public void onCallback(final JSONObject options) {
-        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })($wnd,$doc,'script','//www.google-analytics.com/analytics.js','__ua');
+    private void init() {
+        nativeInit();
+        if (!fallbackPath.isEmpty()) {
+            fallbackInitTimer.schedule(10000);
+        }
+    }
+
+    public void fallback(JsArrayMixed arguments) {
+        if ("send".equals(arguments.getString(0))) {
+            JSONObject jsonOptions = new JSONObject(arguments.getObject(arguments.length() - 1));
+            StringBuilder url = new StringBuilder();
+            url.append(fallbackPath).append("?");
+            url.append(ProtocolTranslator.getFieldName("hitType")).append("=")
+                    .append(URL.encodeQueryString(arguments.getString(1)));
+
+            for (String key : jsonOptions.keySet()) {
+                if (!"hitCallback".equals(key)) {
+                    JSONValue jsonValue = jsonOptions.get(key);
+                    String strValue = "";
+                    if (jsonValue.isBoolean() != null) {
+                        strValue = jsonValue.isBoolean().booleanValue() + "";
+                    } else if (jsonValue.isNumber() != null) {
+                        strValue = jsonValue.isNumber().doubleValue() + "";
+                    } else if (jsonValue.isString() != null) {
+                        strValue = jsonValue.isString().stringValue();
+                    }
+                    url.append("&").append(ProtocolTranslator.getFieldName(key)).append("=")
+                            .append(URL.encodeQueryString(strValue));
+                }
+            }
+            try {
+                RequestBuilder requestBuilder = new RequestBuilder(RequestBuilder.GET,
+                        url.toString());
+                requestBuilder.setCallback(new RequestCallback() {
+
+                    @Override
+                    public void onResponseReceived(Request request, Response response) {
+                        // TODO call hitcallback if needed.
+                    }
+
+                    @Override
+                    public void onError(Request request, Throwable exception) {
+                        // TODO Auto-generated method stub
+                    }
+                });
+                requestBuilder.send();
+            } catch (RequestException e) {
+            }
+        }
+    }
+
+    private native boolean loadFallback() /*-{
+        var name = $wnd['GoogleAnalyticsObject'];
+        var preCalled = $wnd[name].q;
+        if (preCalled !== undefined) {
+            var that = this;
+            $wnd[name] = function() {
+                $entry(that.@com.arcbees.analytics.client
+                    .ClientAnalytics::fallback(Lcom/google/gwt/core/client/JsArrayMixed;)(arguments));
```

Comment:
@meriouma, any way to turn off the checkstyle linelength for this line.

`that.@com.arcbees.analytics.client.ClientAnalytics::fallback(Lcom/google/gwt/core/client/JsArrayMixed;)(arguments)`

Should be on a single line.  The code actually doesn't work as written.